multinet/table.py

```diff
@@ -0,0 +1,109 @@
+"""Operations that deal with tables."""
+from __future__ import annotations  # noqa: T484
+from arango.collection import StandardCollection
+from arango.aql import AQL
+
+from multinet import util
+from multinet.types import EdgeTableProperties
+from multinet.errors import InternalServerError
+
+from typing import List, Set, Dict, Iterable, Union, Optional
+
+
+class Table:
+    """Table."""
+
+    def __init__(self, name: str, workspace: str, handle: StandardCollection, aql: AQL):
+        """Init Table class."""
+        self.name = name
+        self.workspace = workspace
+        self.handle = handle
+        self.aql = aql
+
+    def rows(self, offset: Optional[int] = None, limit: Optional[int] = None) -> Dict:
+        """Return the desired rows in a table."""
+        rows = self.handle.find({}, skip=offset, limit=limit)
+        count = self.handle.all().count()
+
+        return {"count": count, "rows": list(rows)}
+
+    def row(self, doc: Union[Dict, str]) -> Union[Dict, None]:
```

Comment:
If it's a `str`, it can be the document ID or key. https://python-driver-for-arangodb.readthedocs.io/en/master/specs.html#arango.collection.StandardCollection.get

Reply:
Great, thanks for the explanation.

BTW I noticed we can use `Optional` here as well:

```suggestion
    def row(self, doc: Union[Dict, str]) -> Optional[Dict]:
```